### PR TITLE
Add MTU to CNI result

### PR DESCRIPTION
**Pull request by SchSeba**
This commit allow CNIs to expose the MTU of interface

### CODE REVIEW

**Review by bleggett — APPROVED** [review]
There was a good point brought up that we could put this in a metadata struct, but I think MTU is pretty unambiguously a core property of the interface so it makes sense here to me.

Another good point was brought up that if MTU changes, this value could be stale, but I think that's generally true of most things here.

File: SPEC.md [hunk 1]
```diff
@@ -567,6 +567,7 @@ Plugins must output a JSON object with the following keys upon a successful `ADD
 - `interfaces`: An array of all interfaces created by the attachment, including any host-level interfaces:
     - `name` (string): The name of the interface.
     - `mac` (string): The hardware address of the interface (if applicable).
+    - `mtu`: The MTU of the interface (if applicable).
```
Comment (s1061123) [inline SPEC.md:570]: Could you add type in the fields? i.e. `(uint)`

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: SPEC.md (hunk 1 above)
Comment (SchSeba) [reply SPEC.md]: nice catch thanks!

**Review by s1061123 — APPROVED** [review]
/lgtm

**Pull request merged**